leaderelection/chaosduck/main.go

```diff
@@ -117,8 +118,10 @@ func quack(ctx context.Context, kc kubernetes.Interface, component string, leade
 
 func main() {
 	ctx := signals.NewContext()
-	ctx = sharedmain.EnableInjectionOrDie(ctx, nil)
-
+	ctx, informers := sharedmain.EnableInjectionOrDie(ctx, nil)
+	if err := controller.StartInformers(ctx.Done(), informers...); err != nil {
```

Comment:
yeah ^ exactly, I don't want to need to copy this chunk of code in the dozens of places `EnableInjectionOrDie` is used.

Reply:
Same here, will it be better if the `EnableInjectionOrDie` function return a start function instead of `[]informers`, so downstream just need to call a function instead of this chunk of code? I do know this will still break the API.